software-engineer/pt_BR/do-and-donts.md

```diff
@@ -0,0 +1,104 @@
+# Dicas do que fazer e o que não fazer durante as entrevistas
+
+Esta é uma lista direta e destilada de o que fazer e não fazer em entrevistas técnicas, principalmente para entrevistas algorítmicas. Alguns deles podem se aplicar apenas a entrevistas or vídeo ou entrevistas no quadro branco, mas a maioria se aplica a ambos. 
+
+#### Lenda
+
+- ✅ = Faça
+- ❌ = Não
+- ⚠️ = Situacional
+
+### Antes da entrevista
+
+| | Coisas |
+| --- | --- |
+| ✅ | Prepare caneta, papel e fones de ouvido / fones de ouvido. |
+| ✅ | Encontre um ambiente silencioso com boa conexão à Internet. |
```

Comment:
Mixed feelings sobre essa frase. Muitas pessoas não possuem um ambiente silencioso em casa ou uma conexão boa a internet. Tipo, quem tem filho(s) pequenos em casa ou uma obra próxima ou mesmo outras pessoas trabalhando de casa, no mesmo espaço, pode ter dificuldades com encontrar um ambiente silencioso... Da mesma forma, a conexão a internet é a que a pessoa já possui, a gente dá um jeito de seguir se ela não for perfeita (tipo desligando câmeras...)